src/test/java/core/basesyntax/service/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.handler;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final int MORE_ZERO = 5;
+    private static final int ZERO = 0;
+    private static OperationHandler operationHandler;
+    private int actual;
+    private int expected;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandler = new BalanceOperationHandler();
+    }
+
+    @Test
+    public void calculateQuantity_twoOperandsMoreThanZero_isOk() {
+        actual = operationHandler.calculateQuantity(MORE_ZERO, MORE_ZERO);
+        expected = 10;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void calculateQuantity_withZeroOperand_isOk() {
```

Comment:
what if value will be negative?

Reply:
in StoreSerivceImpl I throw new RuntimeException if Quantity below zero, and test it in StoreServiseImplTest with line "p,apple,-1"